NOTICE.txt

```diff
@@ -8,6 +8,48 @@ This document includes a list of open source components used in Mattermost Web A
 
 --------
 
+## @formatjs/intl-pluralrules
+
+This product contains '@formatjs/intl-pluralrules' by FormatJS.
+
+Polyfill for Intl.PluralRules
+
+* HOMEPAGE:
+  * https://github.com/formatjs/formatjs
+
+* LICENSE: MIT License
+
+Copyright (c) 2019 FormatJS
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+---
+
+## @formatjs/intl-relativetimeformat
+
+This product contains '@formatjs/intl-pluralrules' by FormatJS.
+
+`Intl.RelativeTimeFormat`
```

Comment:
Not sure if we can find a description for this.

Reply:
"Polyfill for Intl.RelativeTimeFormat" would work. It's a similar library to the other one.